misc/chip_data.json

```diff
@@ -8379,7 +8379,7 @@
     },
     "6668fd38-076e-4d03-b9a1-02eca5edd0b7": {
       "ReadonlyName": "Trigger Volume Set Filter Tags",
-      "Description": "",
+      "Description": "Sets the filterd tags for the trigger zone",
```

Comment:
stay consistent with the name "trigger volume" instead of "zone". similar to chips v nodes we know people will change out the wording but lets make sure our internal descriptions match up.